### PR TITLE
Ftr 2552 add remove studen attachment files guider

### LINKED ISSUE #2552: Add and remove student attachment files in guider

### PR DESCRIPTION
**Pull request by ilmoeuro**
** Fixes #2552

### CODE REVIEW

File: muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/guider/GuiderRESTService.java [hunk 1]
```diff
@@ -92,4 +106,30 @@ public Response getWorkspaceAssessmentsStudyProgressAnalysis(@PathParam("WORKSPA
...
+    if (ue == null) {
+      return Response.status(Status.NOT_FOUND).entity("User entity not found").build();
+    }
+    
+    List<TranscriptOfRecordsFile> torFiles = torFileController.listFiles(ue);
+    return Response.ok().entity(torFiles).build();
+  }
+
+  @DELETE
+  @Path("/files/{ID}")
```
Comment (anttiviljakainen) [inline muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/guider/GuiderRESTService.java]: Should this be /users/{IDENTIFIER}/files/{ID} ?

File: muikku-core-plugins/src/main/resources/META-INF/resources/dust/guider/guider_view_profile.dust [hunk 2]
```diff
@@ -92,6 +93,29 @@
...
+          </form>
+        </div>
+      </div>
+      <div data-file-list>
+        <div class="gt-user-file flex-row">
+          <div class="gt-user-file-name lg-flex-cell-12 md-flex-cell-12 sm-flex-cell-12">
+            <span data-file-name>filename.tga</span>
+          </div>
+          <div class="gt-user-file-action lg-flex-cell-4 md-flex-cell-4 sm-flex-cell-4 flex-right">
+            <span style="cursor:pointer" data-file-delete>Poista</span>
```
Comment (anttiviljakainen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/dust/guider/guider_view_profile.dust]: Localization?

File: muikku-core-plugins/src/main/resources/META-INF/resources/jsf/records/fileform.xhtml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  </ui:define>
+  
+  <ui:define name="mf-content-master"> 
+    <div class="records">
+      <h2 class="main-functionality-page-title">#{i18n.text['plugin.records.pageTitle']}</h2>
+      <form method="post" enctype="multipart/form-data" action="/transcriptofrecordsfileupload/1">
+        <input type="text" name="title" />
+        <input type="text" name="description" />
+        <input type="file" name="upload" />
+        <button type="submit">Lähetä</button>
```
Comment (anttiviljakainen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/jsf/records/fileform.xhtml]: Localization?

File: muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/guider/GuiderJsPluginMessages_en.properties [hunk 4]
```diff
@@ -78,4 +79,5 @@ plugin.guider.assessmentState.UNASSESSED = Assessment request not sent
 plugin.guider.assessmentState.PENDING = Assessment request has been sent
 plugin.guider.assessmentState.CANCELED = Assessment request has been cancelled
 plugin.guider.assessmentState.PASS = Assessed with a passing grade
-plugin.guider.assessmentState.FAIL = Assessed wit a non-passing grade
\ No newline at end of file
+plugin.guider.assessmentState.FAIL = Assessed wit a non-passing grade
```
Comment (anttiviljakainen) [inline muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/guider/GuiderJsPluginMessages_en.properties]: wit -> with

File: muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/guider/GuiderRESTService.java (hunk 1 above)
Comment (ilmoeuro) [inline muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/guider/GuiderRESTService.java]: This endpoint looks up any file by ID, not by user and ID

**Pull request merged**